spec/models/book_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+describe Book, type: :model do
```

Comment:
We use this format to test validations:   it { is_expected.to validate_presence_of(:name) }

For more information about this style check this repo: https://github.com/thoughtbot/shoulda-matchers#should-vs-is_expectedto